src/bin/bombadil.rs

```diff
@@ -65,6 +66,9 @@ fn main() {
                 .takes_value(true)
                 .multiple(true)
                 .required(false)))
+        .subcommand(SubCommand::with_name(UNLINK))
```

Comment:
`subcommand` closing parenthesis should be after `about`, currently the help message is not displayed correctly.

Reply:
Ahhh...thanks for catching that!